.swiftlint.yml

```diff
@@ -0,0 +1,15 @@
+# Realm Swift's SwiftLint configuration file
+# Uncomment the commented lines to avoid any violations
```

Comment:
I'm not sure what "Uncomment the commented lines" refers to. Is this part of a boilerplate header?